test/fixtures/rules/std_rule_cases/L004.yml

```diff
@@ -26,6 +26,12 @@ spaces_fail_custom_tab_space_size:
 spaces_fail_custom_tab_space_size_not_multiple:
   fail_str: "     \nSELECT 1"
 
+  violations:
+  - description: 'Incorrect indentation type found in file. The number of spaces is
+    not a multiple of '
+    line_no: 1
+    line_pos: 1
+
```

Comment:
Since we usually separate our test cases by two newlines, I always find it really confusing when we also add a newline between sections of a test. Appreciate some of these are existing rather than what you added but Any chance we could clean up the extra white spaces when touching that file? That's what I've been doing to slowly get rid of these.